migrations/update_0_3_3.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ *
+ * @package phpBB Extension - Wiki
+ * @copyright (c) 2016 tas2580 (https://tas2580.net)
+ * @license http://opensource.org/licenses/gpl-2.0.php GNU General Public License v2
+ *
+ */
+
+namespace tas2580\wiki\migrations;
+
+class update_0_3_3 extends \phpbb\db\migration\migration
+{
+	public static function depends_on()
+	{
+		return array(
+			'\tas2580\wiki\migrations\initial_module',
+			'\tas2580\wiki\migrations\update_0_1_2',
+			'\tas2580\wiki\migrations\update_0_2_0',
+			'\tas2580\wiki\migrations\update_0_3_0',
+			'\tas2580\wiki\migrations\update_0_3_1',
+			'\tas2580\wiki\migrations\update_0_3_2',
+		);
+	}
+
+	public function update_schema()
+	{
+		return array(
+			'add_columns' => array(
+				$this->table_prefix . 'wiki_article' => array(
+					'article_time_created'		=> array('TIMESTAMP', 0),
+				),
+			),
```

Comment:
Auch wenn das natürlich mitgelöscht wird wenn man die Erweiterung entfernt würde ich mir hier ein revert_schema für gönnen. Ab phpBB 3.2 kann man nämlich auch einzelne Migrations rückgängig machen (ich habe das mal mit einem "Downgrade" auf 3.1 ausprobiert, funktionierte wunderbar) und das gäbe hier dann Chaos wenn man anschließend die Migration wieder versucht zu installieren.